### DISCUSSION TITLE
Development not working

### DISCUSSION THREAD

**Discussion by AviAvinav** [tamagui/tamagui#171] [Q&A]
I am really new to tamagui & I tried to initiate an app with `npm create tamagui-app@latest`. I installed the dependencies and started the development server with `yarn web` only for it to fail with the following message:

```
➤ YN0000: built @my/ui in 4179 ms
➤ YN0000: Done in 4s 795ms
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
Disabling static extraction in development mode for better HMR
info  - automatically enabled Fast Refresh for 1 custom loader
error - ../packages/app/provider/index.tsx
Module parse failed: Unexpected token (5:46)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import { TamaguiProvider, TamaguiProviderProps } from '@my/ui'
|
> export function Provider({ children, ...rest }: Omit<TamaguiProviderProps, 'config'>) {
|   return (
|     <TamaguiProvider config={config} disableInjectCSS defaultTheme="light" {...rest}>
```

I am not sure what went wrong, please help me out with the same.

**Comment by ibedwi** [comment]
Wanted to help, but I don't know how to reproduce. Just now, tried to initiate a new app using `npm create tamagui-app@latest` and run `yarn web` but everything works perfectly.

Can you try re-initiate the app?

Or if you don't mind, can push your code so I can help investigate further?

**Reply by AviAvinav** [reply]
I have pushed the code I am having problem with in this repo: https://github.com/AviAvinav/tamagui-test-app. I reinitiated the app but the problem persists. Please help me out with the same.

**Reply by ibedwi** [reply]
Hmm, I've tried your repo and run the same command (`yarn web`).
Unfortunately, I can't replicate your issue.

Here's what I did:

1. Clone your repo
2. Install modules, `yarn install`
3. Run `yarn web`

<img width="1680" alt="Screen Shot 2022-09-04 at 17 17 00" src="https://user-images.githubusercontent.com/15979404/188308642-b5d084fd-9bf3-4919-8e9c-342fd07f00e2.png">

Any missing step?

**Reply by AviAvinav** [reply]
That was all that was required to reproduce the issue. I will try it once on WSL too, I was doing it on windows before. It's probably an issue with my machine or node.js version

**Reply by ibedwi** [reply]
Sure! Fyi, I've tried on Node.js `v14.18.2` and `16.14.2`. Both works.

Meanwhile, I've searched for the error on the internet. Most of the answers related to Webpack. Wondering is it caused by it.

**Reply by AviAvinav** [reply]
I tried wsl, and node v16.14.2 but the result remains the same, guess I will read up on Webpack loaders a bit. Thanks for the help.

**Reply by ibedwi** [reply]
Just found out there is related issues https://github.com/tamagui/tamagui/issues/151

**Reply by AviAvinav** [reply]
It seems the problem is not happening with tamagui, I tried to use another monorepo today and I have a similar error about loaders
